app/components/page-components/Chatbot/ChatbotCommands.vue.ts

```diff
@@ -0,0 +1,13 @@
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+
+@Component({
+})
+export default class ChatbotModules extends Vue {
+  data() {
```

Comment:
FYI we generally prefer to use class attributes for reactive data on components.